cypress/support/commands.js

```diff
@@ -107,6 +108,23 @@ const finishedLoading = () => {
   cy.get('[role="progressbar"]').should('not.exist');
 };
 
+const createProposal = (proposalTitle = '', proposalAbstract = '') => {
+  const title = proposalTitle || faker.random.words(3);
```

Comment:
faker.random ofter returns words with punctuations and dash or space-separated words. This can cause failed test false positives so maybe we should favour faker.lorem instead

Reply:
Yes I was thinking about that as well. But we should change it everywhere not only here then.